Climedo doesn't sync correctly (EJBAccessException)

<!--
If you've never submitted an issue to the SORMAS repository before or this is your first time using this template, please read the Contributing guidelines (https://github.com/hzi-braunschweig/SORMAS-Project/blob/development/docs/CONTRIBUTING.md) for an explanation of the information we need you to provide. You don't have to remove this comment or any other comment from this issue as they will automatically be hidden.
-->
### Bug Description
<!-- Mandatory -->
There seems to be a user rights or authentication problem when trying to sync climedo with SORMAS. 
The error in logs says: "Client not authorized for this invocation"

### Steps to Reproduce
<!-- Optional; please add more steps if necessary -->
Register SMS as well as by e-mail, enter the symptoms and submit. 
No error message, but the system does not show the entry, neither under "Follow-up" nor in the "Follow-up view".

Doesn't work with "ExternalVisitDto #personUUID: XXGMVF-AF3BDN-6D2DPY-ECW4KG2I" on https://test1402.sormas.netzlink.com/

### Expected Behavior
<!-- Optional -->

### Screenshots
<!-- Optional -->

### System Details
<!-- Mandatory; you only have to specify the Server URL if the error appeared on a publicly available test server -->
* Device: 
* SORMAS version: 1.70.2
* Android version/Browser: 
* Server URL: 1402
* User Role:

### Additional Information
<!-- Optional -->
Log excerpt [Climedo_User_Not_Authorized.txt](https://github.com/hzi-braunschweig/SORMAS-Project/files/8479129/Climedo_User_Not_Authorized.txt)

```bash
Caused by: javax.ejb.EJBAccessException: Client not authorized for this invocation
	at com.sun.ejb.containers.BaseContainer.mapLocal3xException(BaseContainer.java:2397)
	at com.sun.ejb.containers.BaseContainer.postInvoke(BaseContainer.java:2183)
	at com.sun.ejb.containers.BaseContainer.postInvoke(BaseContainer.java:2104)
	at com.sun.ejb.containers.EJBLocalObjectInvocationHandler.invoke(EJBLocalObjectInvocationHandler.java:220)
	at com.sun.ejb.containers.EJBLocalObjectInvocationHandlerDelegate.invoke(EJBLocalObjectInvocationHandlerDelegate.java:90)
	at com.sun.proxy.$Proxy665.toDto(Unknown Source)
	at de.symeda.sormas.backend.caze.__EJB31_Generated__CaseFacadeEjb$CaseFacadeEjbLocal__Intf____Bean__.toDto(Unknown Source)
	at de.symeda.sormas.backend.visit.VisitFacadeEjb.onVisitChanged(VisitFacadeEjb.java:622)
```

--- a/sormas-backend/src/main/java/de/symeda/sormas/backend/caze/CaseFacadeEjb.java
+++ b/sormas-backend/src/main/java/de/symeda/sormas/backend/caze/CaseFacadeEjb.java
@@ -2610,7 +2610,9 @@ public class CaseFacadeEjb extends AbstractCoreFacadeEjb<Case, CaseDataDto, Case
 		return dto;
 	}
 
-	@RolesAllowed(UserRight._EXTERNAL_VISITS)
+	@RolesAllowed({
+		UserRight._CASE_VIEW,
+		UserRight._EXTERNAL_VISITS })
 	public CaseDataDto toDto(Case source) {
 		return toCaseDto(source);
 	}

--- a/sormas-backend/src/main/java/de/symeda/sormas/backend/caze/CaseFacadeEjb.java
+++ b/sormas-backend/src/main/java/de/symeda/sormas/backend/caze/CaseFacadeEjb.java
@@ -1378,7 +1378,8 @@ public class CaseFacadeEjb extends AbstractCoreFacadeEjb<Case, CaseDataDto, Case
 	@Override
 	@RolesAllowed({
 		UserRight._CASE_CREATE,
-		UserRight._CASE_EDIT })
+		UserRight._CASE_EDIT,
+		UserRight._EXTERNAL_VISITS })
 	public CaseDataDto save(@Valid @NotNull CaseDataDto dto, boolean systemSave) throws ValidationRuntimeException {
 		return save(dto, true, true, true, systemSave);
 	}
@@ -2609,6 +2610,7 @@ public class CaseFacadeEjb extends AbstractCoreFacadeEjb<Case, CaseDataDto, Case
 		return dto;
 	}
 
+	@RolesAllowed(UserRight._EXTERNAL_VISITS)
 	public CaseDataDto toDto(Case source) {
 		return toCaseDto(source);
 	}

--- a/sormas-backend/src/main/java/de/symeda/sormas/backend/contact/ContactFacadeEjb.java
+++ b/sormas-backend/src/main/java/de/symeda/sormas/backend/contact/ContactFacadeEjb.java
@@ -1606,7 +1606,9 @@ public class ContactFacadeEjb
 		return source.toReference();
 	}
 
-	@RolesAllowed(UserRight._EXTERNAL_VISITS)
+	@RolesAllowed({
+		UserRight._CONTACT_VIEW,
+		UserRight._EXTERNAL_VISITS })
 	public ContactDto toDto(Contact source) {
 		return toContactDto(source);
 	}

--- a/sormas-backend/src/main/java/de/symeda/sormas/backend/contact/ContactFacadeEjb.java
+++ b/sormas-backend/src/main/java/de/symeda/sormas/backend/contact/ContactFacadeEjb.java
@@ -1606,6 +1606,7 @@ public class ContactFacadeEjb
 		return source.toReference();
 	}
 
+	@RolesAllowed(UserRight._EXTERNAL_VISITS)
 	public ContactDto toDto(Contact source) {
 		return toContactDto(source);
 	}

